src/build/packages.rs

```diff
@@ -746,23 +773,23 @@ pub fn validate_packages_dependencies(packages: &AHashMap<String, Package>) -> b
         ]
         .iter()
         .for_each(|(dependency_type, dependencies)| {
-            if let Some(unallowed_dependency_name) = get_unallowed_dependents(packages, package_name, dependencies) {
-                let empty_unallowed_deps = UnallowedDependency{
-                   bs_deps: vec![],
-                   pinned_deps: vec![],
-                   bs_dev_deps: vec![],
+            if let Some(unallowed_dependency_name) =
+                get_unallowed_dependents(packages, package_name, dependencies)
+            {
```

Comment:
I don't know what's up with the formatting. Sometimes it works for me; sometimes, it doesn't.

Reply:
if you have a line that cannot be narrowed, the algorithm chokes